product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -74,8 +76,27 @@ Enabling [Apache Superset](/biganimal/latest/using_cluster/06_analyze_with_super
 
 Enabling [PgBouncer](../getting_started/creating_a_cluster/#pgbouncer) to pool your connections incurs additional costs that depend on your cloud provider. In addition to the cloud provider costs, PgBouncer connects to your primary server and requires an IP address. BigAnimal provisions up to three instances per PgBouncer-enabled cluster to ensure that performance is unaffected, so each availability zone receives its own instance of PgBouncer. The extra VM costs are the 2vcpu SKU times the number of PgBouncer instances. For AWS, the instance type is c5.large. For Azure, the instance type is F2s_v2.
 
-## Billing
+## Payments and billing
+
+Your payment and billing options include:
+
+- Digital self-service using a credit card
+- Direct purchase using the Sales Order Form 
+- Azure Marketplace
+
+### Digital self-service
+You can pay for BigAnimal month-to-month with a credit card by making an [EDB account](https://www.enterprisedb.com/accounts/register). BigAnimal sends you monthly invoices to your email. This invoice includes database costs, and if you're using BigAnimal's cloud, infrastructure costs. 
+
+!!! Note 
+    If you want to pay for a longer term of BigAnimal, like for a year or two, then please contact [Sales](https://info.enterprisedb.com/EDB-Contact-Us.html). 
+
+On the [Billing](https://portal.biganimal.com/billing) page, you can manage your payment methods and view your invoice total. To cancel your monthly subscription immediately, you can select **Cancel purchase**. Or, to cancel your subscription at the next renewal date, select **Cancel auto-renewal**.
```

Comment:
BigAnimal only offers "cancel immediately" semantic. And this function is done by the below steps:

1. login as "EDB Account" 
2. access https://portal.biganimal.com/billing 
3. click "payment method"
4. in the newly opened tab, click "cancel plan". 
<img width="1218" alt="image" src="https://github.com/EnterpriseDB/docs/assets/79249661/b74af736-42ac-4e31-8649-eccaed39c0e4">

<img width="1428" alt="image" src="https://github.com/EnterpriseDB/docs/assets/79249661/514ecacf-747f-4ccc-9026-8c3d63826ab4">

 

There is no "cancel auto-renewal" design.